### PR TITLE
Edit function _get_type.

### PR DESCRIPTION
**Pull request by Omnivanitate**
The one edit is function _get_type. Before await function, call this function will return a Coroutine. The Corutine does not have a "type" attribute. is_file function doesn't work. I change that and add one test.

### CODE REVIEW

File: tests/yadisk_test.py [hunk 1]
```diff
@@ -307,3 +307,26 @@ async def test_get_operation_status_request_url(self):
             "https://asd8iaysd89asdgiu")
         self.assertTrue(is_operation_link(request.url))
         self.assertTrue(request.url.startswith("https://"))
+
+    @async_test
+    async def test_is_file(self):
+
```
Comment (ivknv) [inline tests/yadisk_test.py]: Reference the issue/PR associated with the added test

```suggestion
        # See https://github.com/ivknv/yadisk-async/pull/6

```

File: tests/yadisk_test.py [hunk 2]
```diff
@@ -307,3 +307,26 @@ async def test_get_operation_status_request_url(self):
...
+        buf1 = BytesIO()
+        buf2 = tempfile.NamedTemporaryFile("w+b")
+
+        orig_close = buf1.close
+
+        def wrapper():
+            raise BaseException("WHERERA")
+            orig_close()
+
+        buf1.close = wrapper
```
Comment (ivknv) [inline tests/yadisk_test.py]: These lines don't do anything useful, you should remove them
```suggestion
```

**Review by ivknv — CHANGES_REQUESTED** [review]
Thank you for the PR! There are a few odd lines in the test, you should fix them, I'll merge the PR after that.

File: tests/yadisk_test.py (hunk 2 above)
Comment (Omnivanitate) [reply tests/yadisk_test.py]: Thank you! You're right. I have to be a little more careful.

**Review by Omnivanitate — COMMENTED** [review]
Edit files

**Review by ivknv — APPROVED** [review]
Thanks! Everything looks good now. The new version of the library with the fix will most likely be available tomorrow.

**Pull request merged**